rosetta-source/src/main/resources/cdm-sample-files/functions/business-event/quantity-change/increase-equity-swap-func-input.json

```diff
@@ -3,6 +3,26 @@
     "primitiveInstruction" : {
```

Comment:
Are these changes expected?  I thought the original increase use case was unchanged?